### PR TITLE
feat: added md not completed

### PR DESCRIPTION
**Pull request by J-Ben**
#1 
We try some non certains feature

### CODE REVIEW

**Comment by Marlyse22** [comment]
Please change the content

File: pr_not_okay/not_okay.md
Comment (Marlyse22) [inline pr_not_okay/not_okay.md:1]: Add more elements

**Review by Marlyse22 — CHANGES_REQUESTED** [review]

File: pr_not_okay/not_okay.md
Comment (J-Ben) [reply pr_not_okay/not_okay.md:1]: okay it's done

**Review by Marlyse22 — APPROVED** [review]
Everything is okay

**Pull request merged**